pkg/hubble/build/version.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Go related GitHub action is currently failing with:

```
contrib/scripts/check-missing-tags-in-tests.sh
Test file(s) does not contain a tag privileged_tests or !privileged_tests tags:
make: *** [precheck] Error 1
./pkg/hubble/build/version_test.go
Error: Makefile:510: recipe for target 'precheck' failed
Error: Process completed with exit code 2.
```

see https://github.com/cilium/cilium/pull/13979/checks?check_run_id=1424490543

I think this needs `// +build !privileged_tests`

Reply:
Ah dang it, I always run `go test` locally on the package and forget about this. Thanks, will update the PR.